### DISCUSSION TITLE
classification via inference on LLaVA?

### DISCUSSION THREAD

**Discussion by flab305** [haotian-liu/LLaVA#1640] [Q&A]
I am new to the LLaVA framework. I have a dataset of image-text pair classified into 2 classes. Using inference on a trained LLaVA model, I want to get classification on my dataset, i.e. given text and image, classify into one of two labels.

Is there a way to do so? like an inbuilt function or adding MLP head over (getting logits or after fine-tuning etc...)?

**Comment by midofalasol** [comment]
I have the same problem. Have you solved it? @flab305
